utils/uicomponents/src/main/java/dev/xxxxx/uicomponents/TitleValueTextView.kt

```diff
@@ -0,0 +1,44 @@
+package dev.xxxxx.uicomponents
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import dev.xxxxx.uicomponents.databinding.TitleValueTextviewLayoutBinding
+
+class TitleValueTextView: LinearLayout {
+
+    var valueText: String = ""
+        set(value) {
+            field = value
+            binding.tvValue.text = field
+            invalidate()
+            requestLayout()
+        }
+
+    var titleText: String = ""
+        set(value) {
+            field = value
+            binding.tvTitle.text = field
+            invalidate()
+            requestLayout()
+        }
+
+    private val binding = TitleValueTextviewLayoutBinding.inflate(LayoutInflater.from(context), this, true)
+
+    constructor(context: Context): this(context, null)
+    constructor(context: Context, attributeSet: AttributeSet?): super(context, attributeSet, 0){
+        setUp(attributeSet)
+    }
```

Comment:
¿No son 3 constructores los obligatorios?

Reply:
Sólo son obligatorios los dos primeros: 
https://medium.com/@Zielony/guide-to-android-custom-views-constructors-df47476e334c